megamek/src/megamek/common/AmmoType.java

```diff
@@ -537,6 +539,21 @@ public EnumSet<Munitions> getMunitionType() {
         return munitionType;
     }
 
+    public boolean notAllowedByClanRules() {
```

Comment:
This will probably need to be incorporated into the ammo auto-config validity check.  I'll take a look.

Reply:
It was in two locations so just made a helper method for it. Seemed better to put it here vs elsewhere. It should be able to tell us if it's valid and gives us a single place to work with.